modules/core/src/main/scala/fs2/kafka/CommittableProducerRecords.scala

```diff
@@ -34,26 +35,21 @@ import fs2.kafka.internal.syntax._
   * - `CommittableProducerRecords#one` to produce exactly one record within
   * the same transaction as the offset is committed.
   */
-sealed abstract class CommittableProducerRecords[F[_], G[+_], +K, +V] {
```

Comment:
Trying to support `G[_]` here is a major pain when trying to batch records into transactions. The value-add seems to be allowing users to build instances of this class from semi-arbitrary collections of records. We can get the same value by keeping `G[_]` in the static builder methods, but then convert to `Chunk` right away to make downstream processing simpler.

Reply:
Sounds good! 👍